### PR TITLE
feat(stylelint): Add alpha-value-notation rule to stylelint

### PR DESCRIPTION
**Pull request by cgilroy**
## Motivations
It was discovered that when CSS is packaged/optimized in React, percentage-based opacities end up as 1%, no matter the actual value (see bug ticket [here](https://jobber.atlassian.net/browse/GROW-2474?atlOrigin=eyJpIjoiYTM3Y2IxMDJkZGIwNGJkMmIyMDUwNjM4YWZlMmIyYzciLCJwIjoiaiJ9)).  We are adding this rule to stop this issue from happening in the future.

### Added
- added the `alpha-value-notation` rule to the stylelint config
- 

### Changed
- updated `stylelint` to 14.2.0
- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing
- pull the branch, run an `npm ci` to update packages
- running `npm run lint` should complete with no errors

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,14 +79,15 @@
     "postcss-import": "^12.0.1",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.7.0",
+    "postcss-safe-parser": "^4.0.2",
```
Comment (eddysims) [inline package.json:82]: What is this one added for?

File: package.json [hunk 2]
```diff
@@ -79,14 +79,15 @@
...
     "prettier": "^2.4.1",
     "prop-types": "^15.7.2",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-helmet-async": "^1.0.4",
     "react-router-dom": "^5.2.0",
     "style-loader": "^0.23.1",
-    "stylelint": "^11.1.1",
+    "stylelint": "^14.2.0",
```
Comment (eddysims) [inline package.json:90]: What is the reason that we needed to upgrade?

**Review by eddysims — CHANGES_REQUESTED** [review]
@cgilroy just a couple small questions here. Nothing major

File: package.json (hunk 1 above)
Comment (cgilroy) [reply package.json]: Oops, this one snuck in when it shouldn't have!

File: package.json (hunk 2 above)
Comment (cgilroy) [reply package.json:89]: `stylelint` didn't add the `alpha-value-notation` rule until [13.10.0](https://stylelint.io/changelog/#13100)

File: packages/stylelint-config/stylelint.config.js [hunk 3]
```diff
@@ -12,5 +12,6 @@ module.exports = {
     "color-format/format": { format: "rgb" },
     "plugin/declaration-block-no-ignored-properties": true,
     "order/properties-order": require("./properties-order"),
+    "alpha-value-notation": "number",
```
Comment (darryltec) [inline packages/stylelint-config/stylelint.config.js:15]: Also need to update the peer dependency to 14.2.0 under the `packages/stylelint-config/package.json` so this works on ~jobber online as well~ any project that uses `@jobber/stylelint`. _Which, I just found out that isn't really used in Jobber Online_ 😨 

https://github.com/GetJobber/atlantis/blob/1146169ab41ee01106d292f8722ef498f581d629/packages/stylelint-config/package.json#L16-L18

**Review by darryltec — CHANGES_REQUESTED** [review]

**Review by darryltec — APPROVED** [review]
LGTM! Sorry this took a while

**Pull request merged**